bears/go/GoVetBear.py

```diff
@@ -25,5 +27,93 @@ class GoVetBear:
     CAN_DETECT = {'Unused Code', 'Smell', 'Unreachable Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
-        return 'vet', filename
+    def create_arguments(filename, file, config_file,
+                         disallow_assembly_go_mismatches: bool=False,
```

Comment:
If every check is enabled by default by go vet, then all of these need to be defaulting to `True` to replicate existing behaviour of the bear.  Otherwise, checks which were enabled before your change will be deactivated by your change, leaving a repository with no active checking.

Reply:
Quoting from the docs:
>By default the -all flag is set so all checks are performed. If any flags are explicitly set to true, only those tests are run. Conversely, if any flag is explicitly set to false, only those tests are disabled. Thus -printf=true runs the printf check, -printf=false runs all checks except the printf check.